docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+
+Now let's spin up Hardhat's [developer console](https://hardhat.org/guides/hardhat-console.html) to start interacting with our `Filet` contract:
+
+```shell
+npx hardhat console --network local
+> Welcome to Node.js v14.18.1.
+> Type ".help" for more information.
+> >
+```
+
+Now, type the following into the prompt to instantialize the contract object:
+
+```js
+>> const Filet = await ethers.getContractFactory("Filet")
+>> const filet = await Filet.attach("0x5FbDB2315678afecb367f032d93F642f64180aa3")
+```
+
+Next, we inspect the accounts and the balances:
+
+```js
+>> const accounts = await ethers.provider.listAccounts()
+>> accounts
+> [
+>   '0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC',
+>   '0x9632a79656af553F58738B0FB750320158495942',
+>   '0x55ee05dF718f1a5C1441e76190EB1a19eE2C9430',
+>   // ...
+> ]
+```
+
+The array, unsurprisingly, should contain all the addresses listed with the previous `npx hardhat accounts` command. Select one of the addresses to inspect the balance (In this case, we are choosing the second account listed)
+
+```js
+>> const balance = (await filet.balanceOf(accounts[1])).toString()
+> 0
+```
+
+Obviously `0x9632a79656af553F58738B0FB750320158495942` does not own any FIT token yet. Let's mint some and send to the address with:
+
+```js
+>> const tokenId = await filet.callStatic.mintTo(accounts[1], "ipfs://bafkreigaymo3qz73w4nit2matfs7dugczda5wuwzq4g3o2chz4f6nugtaq/metadata.json")
+>> tokenId.toString()
+> '1'
+>> const bal = (await filet.balanceOf(account[1])).toString()
+>> bal
+> '1'
+```
+
+>> 💡 **Why calling `callStatic`?**
+>> The return value of a non-pure or -view Solidity function is available only when the function is called on-chain (from the same contract or another contract).  
+>> When such function is called off-chain (i.e. from ethers.js as we are), the return value is the hash of that transaction. If we called `mintTo` directly, we would receive in return the transaction object, not the token ID we expect from the actual `mintTo` method in the contract.  
+>> To learn more, read [View and Pure Functions](https://solidity-by-example.org/view-and-pure-functions/), this [Stack Exchange post](https://ethereum.stackexchange.com/questions/88119/i-see-no-way-to-obtain-the-return-value-of-a-non-view-function-ethers-js), and [ethers.js doc for `callStatic`](https://docs.ethers.io/v5/single-page/#/v5/api/contract/contract/-%23-contract-callStatic).
+
+The receiving address now owns 1 FIT. By the default `ethers` use the first address as the signer of the transaction, therefore it is `0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC` signing off the minting.
+
+Note the `ipfs://...` provided as the token metadata URI. `ERC721URIStorage` is a special implementation of the IERC721 interface with extra capability of setting and getting metadata URI for the token. This token URI will be retrieved by uploading the intial metadata to [nft.storage](https://nft.storage).
```

Comment:
`ERC721URIStorage`[https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721URIStorage] is a special implementation of the IERC721 interface with extra capability to set and get metadata URI for each token